### ISSUE TITLE
JoistButton `super` followed immediately by `mutate`?

### ISSUE THREAD

**Issue by pixelzoom** [phetsims/joist#824]
In JoistButton.ts:

```typescript
    super();

    // We want to mutate eagerly, but must do so after initializing Voicing properties
    this.mutate( options );
```

Is it really necessary to call `super` without options, _immediately_ followed by `mutate`?  

If so... It should be documented, because it's really confusing, and really odd.  And I would definitely reconsider whatever design decision caused this to be necessary.

**Comment by zepumph** [comment]
This is vestigial, and the hackary is no longer needed since updating our mixins to Typescript. The options can be passed directly to super. Let me know if you see any other cases like this!

**Comment by pixelzoom** [comment]
👍🏻 Closing.
